src/sample.rs

```diff
@@ -44,8 +44,24 @@ async fn poll_for_user_input(client_keys: Keys, tx: futures_channel::mpsc::Unbou
 		let mut words = line.split_whitespace();
 		if let Some(word) = words.next() {
 			match word {
-				//TODO: implement help command
-				"help" => { println!("help command - to be implemented"); },
+				"help" => { 
+					println!("\nUsage: <command> [args]\n");
+					println!("\nAvailable commands (See Nostr NIPs for more details):");
+					println!("  help\tShow a list of available commands");
+					println!("  sendtextnote\tSend a text note to the relay");
+					println!("    args: <content> - String of the text sent to the relay");
+					println!("  setmetadata\tSet the metadata for this client");
+					println!("    args: <username> <about> <picture> - The client's metadata strings to set");
+					println!("  recommendserver\tRecommend a server to the relay");
+					println!("    args: <urlrelay> - The url string of the server to recommend");
+					println!("  opensubscription\tOpen a subscription to the relay");
+					println!("    args: <subscriptionid> <kinds> <since> <until> - The metadata strings of the subscription to open");
+					println!("  closesubscription\tClose a subscription to the relay");
+					println!("    args: <subscriptionid> - The subscription to close");
+					println!("  shutdown\tShutdown the relay");
```

Comment:
minor correction: this doesn't shutdown the relay, but instead closes the connection that the client (`civkit-sample`) has with the relay.

Reply:
Yes the `shutdown` is shutting down the [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop) that `civkit-sample` aims to.

Note, the design goal of the sample is to stay _relatively_ simple as a tool for developers and beta testers.